### PR TITLE
Small fixes after Recipes merge

### PR DESCRIPTION
**Pull request by aarontp**
The important fix is the path changes in the plaso/psort tasks.  The other changes are mostly small fixes/nits.

### CODE REVIEW

File: turbinia/turbiniactl.py [hunk 1]
```diff
@@ -30,7 +30,6 @@
 from turbinia import config
 from turbinia import TurbiniaException
 from turbinia.config import logger
-from turbinia.lib import google_cloud
```
Comment (alimez) [inline turbinia/turbiniactl.py:33]: This gets called here https://github.com/google/turbinia/blob/08ce6c7ea6d9be68f478d5db1b4605f0dea1a1f7/turbinia/turbiniactl.py#L455. We should keep it or move it to the if statement

**Review by alimez — COMMENTED** [review]
Just a small problem. Otherwise, lgtm

**Review by alimez — APPROVED** [review]
Lgtm

**Comment by aarontp** [comment]
Not sure why, but the rebasing seems to have messed up the merge.  Starting a new PR.

**Pull request closed without merging**